src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -8,6 +8,30 @@ namespace System.Text.Json.Serialization.Tests
 {
     public static partial class OptionsTests
     {
+        [Fact]
+        public static void SetOptionsFail()
+        {
+            var options = new JsonSerializerOptions();
```

Comment:
Do we have a test for null/default options?

Reply:
Not sure if you mean for just these settings. Most of the tests use the default options, which are immutable because they are not accessible.